esp/esp/program/modules/handlers/onsiteprintschedules.py

```diff
@@ -56,7 +56,7 @@ def module_properties(cls):
             }
 
     @main_call
-#    @needs_onsite
+    @needs_onsite
```

Comment:
Did you test that printing schedules still works?  It wouldn't surprise me if we turned this off because the script wasn't *actually* doing login correctly.  (Although perhaps fixing this issue is more important than keeping `poll_schedules.sh` working, in which case just make sure to make an issue if you break it.)